### PR TITLE
Added Content component

### PR DESCRIPTION
**Pull request by jepsar**
First PR for https://github.com/primefaces-extensions/primefaces-extensions.github.com/issues/797

Showcase will follow.

### CODE REVIEW

File: src/main/java/org/primefaces/extensions/component/content/ContentRenderer.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    protected void encodeMarkup(final FacesContext context, final Content content) throws IOException {
+        String value = Files.readString(resolve(context, content));
+        if (content.isEvalEl()) {
+            value = evaluateEl(context, value);
+        }
+        if (content.isEscape()) {
+            value = value
+                        .replace("<", "&lt;")
+                        .replace(">", "&gt;");
```
Comment (melloware) [inline src/main/java/org/primefaces/extensions/component/content/ContentRenderer.java]: I think you can use EscapeUtils.escapeXML() which should be a PrimeFaces class.

File: src/main/java/org/primefaces/extensions/config/PrimeExtensionsEnvironment.java [hunk 2]
```diff
@@ -59,6 +61,10 @@ public static void setCurrentInstance(PrimeExtensionsEnvironment pfeEnv, FacesCo
         }
     }
 
+    public boolean isCommonmarkAvailable() {
```
Comment (melloware) [inline src/main/java/org/primefaces/extensions/config/PrimeExtensionsEnvironment.java:64]: Maybe just call this `isMarkdownAvailable`

File: src/main/java/org/primefaces/extensions/component/content/Content.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+import org.primefaces.util.LocaleUtils;
+
+/**
+ * <code>Content</code> component.
+ *
+ * @author Jasper de Vries &lt;jepsar@gmail.com&gt;
+ * @since 8.0.3
+ */
+public class Content extends UIComponentBase {
+
```
Comment (melloware) [inline src/main/java/org/primefaces/extensions/component/content/Content.java:30]: I like everything about this except i am not sure of the name `Content` as too generic.  Maybe `StaticContent`??? @Rapster what are your thoughts?

**Review by melloware — CHANGES_REQUESTED** [review]

File: src/main/java/org/primefaces/extensions/config/PrimeExtensionsEnvironment.java (hunk 2 above)
Comment (jepsar) [reply src/main/java/org/primefaces/extensions/config/PrimeExtensionsEnvironment.java:64]: I opted to use the library name.

File: src/main/java/org/primefaces/extensions/component/content/Content.java (hunk 3 above)
Comment (melloware) [reply src/main/java/org/primefaces/extensions/component/content/Content.java:30]: Instead of `Content` maybe `LocalizedContent`?

Comment (Rapster) [reply src/main/java/org/primefaces/extensions/component/content/Content.java:30]: I'm not sure I understand exactly what the component does... By the sound of this:
> We have several applications where we need to output larger blocks of HTML text which can differ per context / country, which we have multiple languages of and can fallback to a default context and language. Message bundles just was not the tool for the job. As I could not find any existing tools I decided to create a component.

It sounds like a specific component is not necessary, but again I don't fully understand the context, and how markdown comes into the picture? Maybe if you could provide a jar for the showcase so we can have a quick look, that would be nice :wink:

**Comment by tandraschko** [comment]
Currently also a -0.5 from my side.
Looks like it can be done with a simple controller (with only some lines of code) in your webapplication + h:outputText escape=false.

**Pull request closed without merging**